1.fee_bumping/fee_bumping.md

```diff
@@ -193,6 +214,20 @@ TODO
 
 TODO
 
+## User Experience Recommendations
+
+Wallets and explorers should present relevant information about transaction
+packages to users:
+
+- if an unconfirmed transaction is part of a package of unconfirmed
+  transactions, the service should present ancestor and descendant feerate of
+  the transaction alongside its feerate, to allow users to more accurately
```

Comment:
I think this is contingent on who the user is. I can imagine someone from a large consumer-facing Bitcoin exchange reading this and saying, "show grandma two feerates, one for the package? yeah right." 

I agree this is information I'd like to see (or at least have accessible), but for the purpose establishing credibility with readers, I'd say you should consider qualifying the recommendation a little. Something like "allow expert users to show a display of the package feerate."

Reply:
Yes, very good point. I'll update it.